ROADMAP.md

```diff
@@ -151,6 +151,11 @@ Any dates listed below and the specific issues that will ship in a given milesto
   * Visible error messages for all error cases surfaced in claims and/or eventing
   * Static provisioning examples to highlight simplicity. 
 
+ * v1beta2 Services APIs
+   * Incorporate beta1 feedback
+   * Upgrade other supported services to v1beta2 (e.g. Buckets, etc.)
```

Comment:
I feel like it would be more in line with Kubernetes versioning semantics to bump these to `v1beta1` (compared to their current `v1alpha3`) rather than `v1beta2`. Or, put otherwise, they should be versioned in relation to themselves, rather than in relation to other `v1betaX` APIs.

Reply:
good point, fixed.